packages/shared/components/SubjectBox.svelte

```diff
@@ -3,15 +3,20 @@
     import { Subject } from '@core/wallet'
     import { Box, AddressBox, Text, AccountLabel, TextType, FontWeight } from 'shared/components'
 
-    export let subject: Subject = null
+    export let subject: Subject | null = null
+
+    enum SubjectType {
+        Account = 'account',
+        Address = 'address',
+    }
```

Comment:
This enum can be exported and used in `account-subject.interface.ts` and `address-subject.interface.ts`

Reply:
I have removed it and will open an issue to replace it when we clean up `lib` :pray: